scripts/uploadDataFiles.js

```diff
@@ -1,41 +1,34 @@
 const fs = require('fs')
-const s3 = require('s3-client')
+const aws = require('aws-sdk')
 const process = require('process')
 
 const splitVersion = process.env.npm_package_version.split('.')
 splitVersion.splice(2)
 const dataFileVersion = splitVersion.join('.')
 
-const client = s3.createClient({
-  maxAsyncS3: 20,
-  s3RetryCount: 3,
-  s3RetryDelay: 1000,
-  multipartUploadThreshold: 20971520,
-  multipartUploadSize: 15728640,
-  // See: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html#constructor-property
-  s3Options: {}
+const client = new aws.S3({
+  maxRetries: 3,
```

Comment:
I wasn't able to find all of the old options since the API has changed significantly. It's not clear to me that they are needed though.